HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/HardcodedInvoices/HardcodedInvoices.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  HardcodedInvoicesController.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+
+protocol HardcodedInvoicesControllerProtocol: AnyObject {
+    func obtainInvoicePhotosHardcoded(completion: @escaping (([Data]) -> Void))
+    func storeInvoicesWithExtractions(invoices: [DocumentWithExtractions])
+    func getInvoicesWithExtractions() -> [DocumentWithExtractions]
+    func appendInvoiceWithExtractions(invoice: DocumentWithExtractions)
+}
+
+final class HardcodedInvoicesController: HardcodedInvoicesControllerProtocol {
+    func obtainInvoicePhotosHardcoded(completion: @escaping (([Data]) -> Void)) {
+        var invoicesData: [Data] = []
+        for i in 1 ... Constants.numberOfInovices {
+            let invoiceTitle = "\(Constants.invoiceTitle)\(i)"
+            if let fileURL = Bundle.main.url(forResource: invoiceTitle, withExtension: Constants.invoiceFileFormat) {
+                do {
+                    let invoiceData = try Data(contentsOf: fileURL)
+                    invoicesData.append(invoiceData)
+                } catch {
+                    print("❌ Couldn't load data from \(invoiceTitle). Error: \(error.localizedDescription)")
+                }
+            } else {
+                print("❌ Invoice with name \(invoiceTitle) doesn't exist.")
+            }
+        }
+        print("✅ Successfully obtained \(invoicesData.count) invoices data")
+        completion(invoicesData)
+    }
+    
+    
+    func storeInvoicesWithExtractions(invoices: [DocumentWithExtractions]) {
+        do {
+            let encoder = JSONEncoder()
+            let data = try encoder.encode(invoices)
+            UserDefaults.standard.set(data, forKey: Constants.invoicesStoredKey)
+            print("✅ Successfully stored invoices in UserDefaults")
+        } catch {
+            print("❌ Unable to Encode Invoices: (\(error))")
+        }
+    }
+    
+    func getInvoicesWithExtractions() -> [DocumentWithExtractions]  {
+        if let data = UserDefaults.standard.data(forKey: Constants.invoicesStoredKey) {
```

Comment:
[Clarification]: @razvancapra do we really need to store the invoices in userDefaults or can we just load them from target resources?🤔

Reply:
In IPC-95 there is a request to "For each uploaded invoice store:". 
@a-szotyori can you clarify?

Reply:
@razvancapra @zladzeyka We talked about this with Nadya. All's good.